internal/isoeditor/rhcos_test.go

```diff
@@ -0,0 +1,118 @@
+package isoeditor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultTestOpenShiftVersion = "4.6"
+	defaultTestServiceBaseURL   = "http://1.1.1.1:6000"
```

Comment:
Nitpick but if you're looking for something to use here I typically pick from one of the ranges reserved for documentation and tests ref: https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Good idea. I'll go with 198.51.100.0/24 then.